index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
- [x] The page needs a good descriptive title.
- Suggestion: you can put `Hello world` as a title.